### PR TITLE
fix: Fix relative links in converted documentation

### PR DESCRIPTION
**Pull request by vam-google**

### CODE REVIEW

**Review by miraleung — APPROVED** [review]

File: src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java [hunk 1]
```diff
@@ -195,7 +202,7 @@ private Field schemaToField(Schema sch) {
       valueType = subField.getValueType();
     }
 
-    Field field = new Field(name, valueType, repeated, keyType, description);
+    Field field = new Field(name, valueType, repeated, keyType, sanitizeDescr(description));
```
Comment (vchudnov-g) [inline src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java:205]: This and the previous usage of `sanitizeDexcr` reference the same description, right? So maybe assigned the sanitized description at the top of the method (and s/`description`/`sanitizedDescription`/)

File: src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java [hunk 2]
```diff
@@ -252,20 +259,21 @@ private Field schemaToField(Schema sch) {
 
   private Message constructEnumMessage(
       String name, String description, List<String> enumVals, List<String> enumDescs) {
-    Message enumMessage = new Message(name, false, true, description);
+    Message enumMessage = new Message(name, false, true, sanitizeDescr(description));
```
Comment (vchudnov-g) [inline src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java:262]: Your're passing an un-sanitized description to `constructEnumMessage`, but for the Field constructor, you're passing an already-sanitized description. How about adopting a policy that whenever the description is passed as a parameter, it is passed pre-sanitized? That way you can can sanitize descriptions as soon as you get them from an unsanitized source.

**Review by vchudnov-g — APPROVED** [review]
Suggest tweaking the PR title to "Ensure use of absolute links in..."

**Review by alexander-fenster — APPROVED** [review]

File: src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java (hunk 1 above)
Comment (vam-google) [reply src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java:205]: Yes, we can do it, but doing it the current way lets us keep the sanitization logic consistent (i.e. sanitization method is called right before calling the constructor of a message/field/service/method).

File: src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java (hunk 2 above)
Comment (vam-google) [reply src/main/java/com/google/cloud/discotoproto3converter/proto3/DocumentToProtoConverter.java:262]: We can do it but it is not consistent with the rest of the logic. There is currently no preprocessign in the converter, only post-processing.

**Pull request merged**